src/libs/actions/Report.ts

```diff
@@ -345,6 +346,7 @@ function addActions(reportID: string, text = '', file?: File) {
         commentReportActionID: file && reportCommentAction ? reportCommentAction.reportActionID : null,
         reportComment: reportCommentText,
         file,
+        createdDate: file ? attachmentAction?.created : reportCommentAction?.created,
```

Comment:
Should we call this `clientCreatedTime`?  It's a "time" not a "date"? And also it seems to suggest that we are going to set the `reportAction.created` to this value.  I would love to make it clear that we are not doing that somehow.

Reply:
Let's call is clientCreatedTime 👍 
Let's update from Auth so we don't have to support two params

Reply:
Thanks! Updated